contracts/provider/asconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "extends": "near-sdk-as/asconfig.json",
-  "entry": "assembly/main.ts",
+  "entry": "assembly/index.ts",
```

Comment:
Switching to a single-file setup because we're not using as many custom types or classes.